_posts/2016-06-06-introducing-pecas.markdown

```diff
@@ -32,8 +32,8 @@ good way to detect deviations.
 
 To build this small application we decided to use
 [Rails 4.2](http://rubyonrails.org/) and the
-[Freckle API v2](http://developer.letsfreckle.com/#overview). You can easily
-set it up using [Heroku](http://heroku.com) and your Freckle API key: [https://github.com/ombulabs/pecas#first-time-only](https://github.com/ombulabs/pecas#first-time-only)
+[Noko API v2](https://developer.nokotime.com/). You can easily
+set it up using [Heroku](http://heroku.com) and your Noko API key: [https://github.com/ombulabs/pecas#first-time-only](https://github.com/ombulabs/pecas#first-time-only)
 
-If you like [Freckle](https://letsfreckle.com) as much as we do, feel free to
+If you like [Noko](https://nokotime.com/) as much as we do, feel free to
 **fork Pecas** in [Github](https://github.com/ombulabs): [https://github.com/ombulabs/pecas](https://github.com/ombulabs/pecas)
```

Comment:
@FionaDL Repo is now under the fastruby organization.

Reply:
@erGood call!

> @FionaDL Repo is now under the fastruby organization.

@etagwerker good call!